examples/README.md

```diff
@@ -0,0 +1,191 @@
+
+The above example will have this behavior:
+* If `%{HTTP_ACCEPT}` matches `text/html`, the server will return an HTML document (`ppop.html`)
+* If `%{HTTP_ACCEPT}` matches `text/turtle`, the server will return a Turtle document (`ppop.ttl`)
+
+The syntax of `RewriteCond` directive is:
+```ApacheConf
+RewriteCond TestString CondPattern [Flags]
+```
+
+Where *CondPattern* is a regular expresssion to match pattern in *TestString*.
+
+In this case, *TestString* is `%{HTTP_ACCEPT}` which its value is taken from an `Accept` field in the HTTP request header. The `Accept` field can be a string like this:
+
+```HTTP
+Accept: text/html, application/xhtml+xml, application/xml, image/webp
+```
+
+Each media type will be presented, separated by a comma. With that, we can use *CondPattern* to matches media types in this string.
+
+
+#### Example 3: Dealing with query string
+
+Everything after the question mark (`?`) in the URL, but not that `?` itself, is a query string.
+
+For example, for the URL `https://en.wikipedia.org/w/index.php?title=Web`, the query string is `title=Web`.
+
+As the query string is not included in the string that the *Pattern* of *RewriteRule* will compared against, you cannot use *Pattern* to match them.
+
+To find pattern in the query string, use `%{QUERY_STRING}` as a *TestString* in *RewriteCond*.
+
+As an example, if you like to redirect the URL `https://w3id.org/example?a=1&b=2`  to `https://example.com/path/file.php?a=1&b=2`, you can use this set of rules:
+
+```ApacheConf
+RewriteCond %{QUERY_STRING} (.*)
+RewriteRule ^ https://example.com/path/file.php?%1? [R=302,L]
+```
+
+In this example, the *CondPattern* `(.*)` in *RewriteCond* will match every characters in the query string (`a=1&b=2`) and put it in group one. This group one (`%1`) is later used in the *Substitution* of *RewriteRule*.
+
+
+
+Note that the special character to recall groups from *CondPattern* of *RewriteCond* is `%` (unlike the special character to recall groups from *Pattern* of *RewriteRule*, which is `$`).
+
+The character `?` at the end of the *Substitution* of *RewriteRule* tells the server not to pass the query string to the final URL after rewrite.
+
+
+## README.md
+
+Each ID hosted on W3ID is expected to have a file named `README.md` containing an information about the ID itself and an information about the maintainer(s). This can be more elaborate than the information inside `.htaccess`.
+
+The `.md` file extension at the end of the file name indicates that the file uses Markdown markup language for text formatting. You can have tables and images as well, if needed.
+
+GitHub will automatically display the content of a `README.md` to repository visitors.
+
+An example of a good README file: [w3id.org/dggs/README.md](https://github.com/perma-id/w3id.org/blob/master/dggs/README.md)
+
+
+## Publish vocabularies with W3ID
+
+If you plan to publish a vocabulary/ontology with W3ID,
+see https://w3id.org/examples/ontology.
```

Comment:
Wouldn't it be better to link to /example/ directly? The one in /examples is a redirect.

Reply:
Thank you. That's better. I have changed that accordingly.